main.go

```diff
@@ -102,57 +109,66 @@ func LoginHandler(w http.ResponseWriter, r *http.Request) {
 	email := d.Email
 	password := d.Password
 
-
 	if email == "" {
-		error.Message = "Email は,必須です。"
+		error := model.Error{}
+		error.Message = "Email は必須です。"
 		errorInResponse(w, http.StatusBadRequest, error)
 		return
 	}
 
 	if password == "" {
-		error.Message = "パスワードは、必須です。"
+		error := model.Error{}
+		error.Message = "パスワードは必須です。"
 		errorInResponse(w, http.StatusBadRequest, error)
 	}
 
 	user.Email = email
 	user.Password = password
 
-
 	db := gormConnect()
 	defer db.Close()
 
-	row := db.Where("email = ?", user.Email).Find(&user2)
-
-	_, err := json.Marshal(row)
-	if err != nil {
+	userData := model.User{}
```

Comment:
空のの構造体作るときは var のがきれいかもしません

```suggestion
	var userData model.User
```